_includes/macros.html

```diff
@@ -238,3 +238,50 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
         <p class="alert_text">{{ text }}</p>
     </div>
 {%- endmacro %}
+
+
+
+{# ==========================================================================
+
+   subscribe()
+
+   ==========================================================================
+
+   Description:
+
+   Creates an email subscription form when given:
+
+   code: A GovDelivery code a specified mailing list
```

Comment:
IDK, it's how the others in this file are formatted. Going to leave it and we can clean it up later if needed.